.github/workflows/e2e-test-optional-periodic.yml

```diff
@@ -0,0 +1,27 @@
+name: Periodic E2E Test Optional
+
+on:
+  schedule:
+    # Run every day at 04:20 UTC (it is recommended to avoid running at the start of the hour)
+    - cron: '20 4 * * *'
+  pull_request:
```

Comment:
This would mean that the periodic would run on PRs as well as we see in the current run, I dont think we want that.

/hold

Reply:
Thanks, good catch! This was in a test commit to make sure that they work. I managed to forget it :sweat: 
Dropping it now an repushing